.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: 'npm'
+    directory: '/'
+    schedule:
+      interval: 'daily'
+    commit-message:
+      prefix: 'fix(CDX-227):'
```

Comment:
I think it should be :

`fix(J:CDX-227)`

for it to be picked up properly by dep pipeline as "disregard this commit + link to JIRA automatically".

Please don't ask my why the `J` is important 😂

Reply:
I reckon the Jira should be compliant with what I have currently. I don't think we need dep pipeline to disregard the dependency commits, just to have the Jira tag (the HTTP link I reckon is unnecessary), be reviewed, and merged through a PR.

Reply:
Found your J, and not needed according to the doc: https://coveord.atlassian.net/wiki/spaces/CM/pages/1786708612/Package+Rollout+Certifier